src/transaction/log_tran_table.c

```diff
@@ -1588,6 +1632,10 @@ logtb_clear_tdes (THREAD_ENTRY * thread_p, LOG_TDES * tdes)
   LSA_SET_NULL (&tdes->rcv.atomic_sysop_start_lsa);
   LSA_SET_NULL (&tdes->rcv.analysis_last_aborted_sysop_lsa);
   LSA_SET_NULL (&tdes->rcv.analysis_last_aborted_sysop_start_lsa);
+
+#if defined (SERVER_MODE)
```

Comment:
Why does this only on SERVER_MODE? It doesn't matter with `csql -S` (Standalone mode)?